packages/material/src/mui-controls/MuiInputText.tsx

```diff
@@ -35,8 +35,24 @@ interface MuiInputTextStatus {
   showAdornment: boolean;
 }
 
+interface MuiTextInputProps {
+  inputProps?: {
```

Comment:
Do we have to re-declare all the props or is there a type exported by MUI that we could re-use?